### PR TITLE
Prepare 1.5.0

### PR DESCRIPTION
**Pull request by jneira**
This one includes #2346, will rebase when it is merged in master 

<a href="https://gitpod.io/#https://github.com/haskell/haskell-language-server/pull/2361"></a>

### CODE REVIEW

File: stack-9.0.1.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-resolver: nightly-2021-09-16
+resolver: nightly-2021-11-06
```
Comment (Ailrun) [inline stack-9.0.1.yaml:1]: I think this `package` should also be fixed.

**Review by Ailrun — APPROVED** [review]
Awesome, I'm eagerly waiting for the full support of GHC 9.0!

**Comment by jneira** [comment]
> 
> 
> Awesome, I'm eagerly waiting for the full support of GHC 9.0!

I am afraid although i uncommented packages we still miss brittany, floskell, tactics and class plugins

File: stack-9.0.1.yaml (hunk 1 above)
Comment (jneira) [reply stack-9.0.1.yaml:1]: I am afraid we cant uncomment packages here cause stack tries to build them even if hls has those plugins disabled.
cabal-install takes them in account for the build plan (so i had to add some allo-newer) but only builds them if they are enabled in hls

**Comment by jneira** [comment]
Run checking the hackage release here: https://github.com/haskell/haskell-language-server/actions/runs/1471669261
As you can observe, it has 9.0.1 disabled :worried:

**Comment by fendor** [comment]
Can we get #2359 also merged?

**Comment by jneira** [comment]
> 
> 
> Can we get #2359 also merged and into this release, or rather not?

I would prefer dogfooding it until the next release, if you are not in a hurry to have the fix

**Comment by fendor** [comment]
Ok, good point!

**Pull request merged**